fftools/ffmpeg.c

```diff
@@ -1770,11 +1786,22 @@ static void print_report(int is_last_report, int64_t timer_start, int64_t cur_ti
         }
         if (!vid && enc->codec_type == AVMEDIA_TYPE_VIDEO) {
             float fps;
-
+/*Proximie*/
+#if 1
+            float cur_fps;
+            tr = ost->fps_reset_time ? (cur_time-ost->fps_reset_time) / 1000000.0 : t;
+            frame_number = ost->total_frames;
+            fps = t > 1 ? frame_number / t : 0;
+            cur_fps = tr > 1 ? ost->frame_number / tr : -1;
+            av_bprintf(&buf, "frame=%5d avgfps=%3.*f fps=%3.*f q=%3.1f ",
+                     frame_number, fps < 9.95, fps, cur_fps>0.0?cur_fps < 9.95:fps < 9.95, cur_fps>0.0?cur_fps:fps, q);
```

Comment:
This looks wrong to me: There's 4 format specifiers but 5 following arguments.
Also, doesn't `fps < 9.95` resolve to a boolean?

Reply:
This is ffmpeg coding... I had to look it up too, the ".*" in the format specifier tells that number of spaces after period is specified in the argument list, which translates to 0 or 1 in this case

Reply:
Fair enough!!